package-res/ccc/core/base/chart/chart.data.js

```diff
@@ -96,6 +116,47 @@ pvc.BaseChart
 
         if(def.debug >= 3) this.log(this.data.getInfo());
     },
+ 
+
+    // CDF603
+    // Auxiliar function to initialize axes prior to the datums distribution
+    _initRolesAxes: function(){
+
+        hasMultiRole = this.visualRoles.multiChart.isBound();
+        this._initAxes(hasMultiRole);
+
+    },
+
+    // CDF603
+    // @virtual
+   _createScoringOptions: function(options) {
```

Comment:
Is there any case where `_createScoringOptions` is overridden and `_createSlidingWindow` is not? It seems they are the same function or, maybe it is that the name of one does not encompass the other, however one calls the other...